docs/roadmap/DSP_roadmap.md

```diff
@@ -0,0 +1,45 @@
+# Dataspace Protocol Specification  - Scope, Roadmap and Milestones
+
+## Scope
+
+The Dataspace Protocol is a set of specifications designed to facilitate interoperable data sharing between entities governed by usage control and based on Web technologies. These specifications define the schemas and protocols required for entities to publish data, negotiate Agreements, and access data as part of a federation of technical systems termed a Dataspace.
+The web rendering of the Dataspace Protocol represents the current state on this repo's main branch: https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/ 
+
+The Dataspace Protocol aims a release as Eclipse Specification combining this specification with the Dataspace TCK (https://projects.eclipse.org/projects/technology.dataspacetck ) and at least one conformant implementation. The Eclipse Dataspace Components EDC (https://projects.eclipse.org/projects/technology.edc ) are planned as compliant implementation, others are welcome. The Dataspace Protocol targets a PAS transposition to ISO during 2025.
```

Comment:
First sentence sounds weird, as the DSP is not combined with other projects.. this is a guideline by EF. Maybe we can link to that. 

Second sentence makes a statement about another project + marketing speech - inappropriate. Would probably rather fit to the project webpage, not the sources (repository).